File: TaskItApi/Entities/Task.cs
using System;
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;

namespace TaskItApi.Entities
{
    public class Task
    {
        [Key]
        [DatabaseGeneratedAttribute(DatabaseGeneratedOption.Identity)]
        public int ID { get; set; }
        [Required]
        public string Title { get; set; }

        public string Description { get; set; }
        public DateTime From { get; set; }
        [Required]
        public DateTime Until { get; set; }
        [Required]
        public Group Group { get; set; }       

    }
}

File: TaskIt.Frontend/src/app/Domain/task.ts
export class Task {
  public id: string | undefined;
  public title: string | undefined;
  public description: string | undefined;
  public endDate: Date | undefined;

  constructor(id: string, title: string, description: string | undefined, endDate: Date | undefined) {
    this.id = id;
    this.title = title;
    this.description = description;
    this.endDate = endDate;
  }
}
